wallet/src/main/scala/org/bitcoins/wallet/DLCWallet.scala

```diff
@@ -260,11 +284,12 @@ abstract class DLCWallet extends Wallet {
         eventId = offer.eventId
       )
 
-      signer = DLCTxSigner(offer = offer,
-                           accept = acceptWithoutSigs,
+      builder = DLCTxBuilder(offer, acceptWithoutSigs)
+
+      signer = DLCTxSigner(builder = builder,
```

Comment:
nit: Feel free to add a constructor to signer that takes in an offer, acceptWithoutSigs and the rest